python/tk_framework_desktopserver/server_protocol.py

```diff
@@ -82,7 +81,7 @@ def onConnect(self, response):
             # response.origin: xyz.shotgunstudio.com
             domain_valid = self._is_domain_valid(response.origin)
         except:
-            self._logger.exception()
+            self._logger.exception("Unexpected error while losing connection.")
```

Comment:
Should this message say something else for clarity? E.g. `Unexpected error while trying to determine the originating domain` or something?